### PR TITLE
Earth data discovery

### PR DESCRIPTION
**Pull request by Marie59**
Just a small introductory training to whats available in the earth system data world. 
It will most likely be updated through the year.

### CODE REVIEW

File: topics/climate/tutorials/earth_system/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+### A focus on ODV (Ocean Data View) collections
+
+This [webODV](https://emodnet-chemistry.webodv.awi.de/) service facilitates to explore, subset, visualize, and extract data sets in multiple formats from the harmonized, standardized, validated data collections that EMODnet Chemistry is regularly producing and publishing for all European sea basins for eutrophication and contaminants. 
+You can also find these collections on the [Sextant catalogue](https://sextant.ifremer.fr/eng/Data/Catalogue#/search?isTemplate=n&from=1&to=30&sortBy=dateStamp&sortOrder=desc&languageStrategy=searchInDetectedLanguage&any=EMODnet%20Chemistry)
+
+You can analize thes kind of data through the [Ocean Data View (ODV)](https://usegalaxy.eu/root?tool_id=interactive_tool_odv) tool. 
+
+For more information follow the related tutorials and workflows:
+ - [Ocean's variables study]({% link topics/climate/tutorials/ocean-variables/tutorial.html %}) and the corresponding workflow  `https://usegalaxy.eu/u/marie.josse/w/oceans-var-v2`
+ - [Ocean Data View (ODV)]({% link https://training.galaxyproject.org/training-material/topics/climate/tutorials/ocean-data-view/tutorial.html %})
```
Comment (hexylena) [inline topics/climate/tutorials/earth_system/tutorial.md:96]:
```suggestion
 - [Ocean Data View (ODV)]({% link topics/climate/tutorials/ocean-data-view/tutorial.md %})
```

File: topics/climate/images/earth_system/earth_system_galaxy.png
Comment (hexylena) [inline topics/climate/images/earth_system/earth_system_galaxy.png:1]: cute! (who/where is the image from? do we have the rights to host this? what's the license?)

File: topics/climate/tutorials/earth_system/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
+---
+layout: tutorial_hands_on
+title: Getting your hands-on earth data
+questions:
+- What is the Earth System?
+- What type of data is available?
+objectives:
+- Learn about the terminology
```
Comment (hexylena) [inline topics/climate/tutorials/earth_system/tutorial.md:8]:
```suggestion
- Learn about the Earth Systems terminology
```

**Review by hexylena — COMMENTED** [review]
Looks good!

File: topics/climate/images/earth_system/earth_system_galaxy.png
Comment (Marie59) [reply topics/climate/images/earth_system/earth_system_galaxy.png:1]: This was made by me on canva ;) so no problem ?

**Comment by Marie59** [comment]
Thanks for the review @hexylena !

File: topics/climate/images/earth_system/earth_system_galaxy.png
Comment (hexylena) [reply topics/climate/images/earth_system/earth_system_galaxy.png:1]: that's sufficient information, thank you :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**